src/frontend/sider/Journalforing.tsx

```diff
@@ -72,6 +77,15 @@ export const Journalforing: React.FC = () => {
 
     useEffect(() => {
         if (journalpostState.innsending.status === RessursStatus.SUKSESS) {
+            const lagredeOppgaveFiltreringer = hentFraLocalStorage(oppgaveRequestKey, {});
+
+            lagreTilLocalStorage(oppgaveRequestKey, {
+                ...lagredeOppgaveFiltreringer,
+                ident:
```

Comment:
jag er inte enig om att man ska lagre identen i localstorage..føles litt risky

Reply:
Vi var også litt skeptiske, men gjorde en vurdering på det sammen med Bent (sikkerhetschampion) og landet på at det var greit. Argumentene for var:

1. Fødselsnummer alene regnes ikke som sensitivt. Vi er mer bekymret for å lekke det i kombinasjon med andre data.
2. Vi regner med at saksbehandlingsløsningen primært skal brukes fra nav/ikke-private maskiner.

Men tar gjerne input på hvorfor vi bør revurdere. Følte som sagt også at det var litt risky tidligere 😊

Reply:
Tolkade det som det var saksbehandlerIdenten som skulle lagras i localstorage .. Er det faktiskt fnr vi lagrer ?

Reply:
Vi vil at saksbehandler etter fullført journalføring skal redirectes til et søk i oppgavebenken på alle oppgaver tilknyttet den personen, derfor setter vi personidenten i localstorage for at vi skal ha den tilgjengelig i oppgavebenken.

Reply:
Intressant tanke ! Är det måten man jobbar på ? Dvs när man journalförer en sak så ska man fortsätta jobba med den personen? Eller ska man fortsätta att journalföra andra saker som har kommit in?

Reply:
Ja, Mirja sa at de journalfører "alt" for en person før de evt. starter å behandle en sak. 

Det er nok todelt arbeidsflyt - de som sitter og "bare" journalfører og de som gjør begge deler. Dette passer for siste gruppa.

Reply:
Jeg har forstått det sånn. Er visstnok vanlig at det er flere oppgaver å journalføre tilknyttet en person, og derfor snakket vi med Mirja om at det er fint at man sendes til et søk på den personen... Men man kan også se for seg at det er irriterende hvis man egentlig er ferdig med den personen 🤔